config/locales/views/contests/hi.yml

```
@@ -131,3 +131,4 @@ hi:
     circuitverse:
       preview_project: "प्रोजेक्ट प्रीव्यू"
   feature_not_available: "प्रतियोगिता फ़ीचर उपलब्ध नहीं है।"
+  feature_not_available: "प्रतियोगिता फ़ीचर उपलब्ध नहीं है।"
```

YAMLlint (1.37.1)

[error] duplication of key "feature_not_available" in mapping

(key-duplicates)